src/main/java/bhs/devilbotz/commands/arm/ArmSafety.java

```diff
@@ -167,5 +172,22 @@ public final boolean isFinished() {
    * execution of the command
    */
   public void initializeWithSafety() {}
-  ;
+
+  private boolean isArmStuck() {
+    // check the rate of change of the arm position
+    // if there is no change, then the rope is broken
+
+    double currentSpeed = Math.abs(arm.getVelocity());
+
+    // start a timer to see if the rope is stuck for a while
+    // if it is, then we need to stop the arm
+    if (currentSpeed < 0.75) {
```

Comment:
The 0.75 should be in ArmConstants.  Unless you already know it, we also need to verify the nominal speed of the arm when everything is working as expected.